obspy/io/mseed/util.py

```diff
@@ -238,31 +275,110 @@ def get_timing_and_data_quality(file_or_file_object):
     info = get_record_information(file_or_file_object)
     # Keep track of the extracted information.
     quality_count = [0, 0, 0, 0, 0, 0, 0, 0]
+    activity_count = [0, 0, 0, 0, 0, 0, 0]
+    io_count = [0, 0, 0, 0, 0, 0]
```

Comment:
I think it is a good idea but then should we do this for all the counts (quality and activity too)?